apps/react/react-2-native.json

```diff
@@ -43,21 +29,10 @@
       "label": "Template",
       "description": "You can find many grea community templates by searching for <a href='https://www.npmjs.com/search?q=expo-template-*'>expo-template-*</a> on npm.",
```

Comment:
Can you please change the description and point to https://github.com/expo/examples url

Reply:
The Filed label may say as Template but you are actually using the examples. 
So The command which generates uses the `-e` which may confuse the user.

Even the official site is recommending the example I suggest we can keep the label as `Example` and change the description link point to official example repos.